autheo/chain.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "autheo",
```

Comment:
testnets require 'testnet' in the chain_name and directory name (e.g., autheotestnet), and must go inside the /testnets/ directory.

Reply:
and must go inside the /testnets/ directory.